.eslintrc.js

```diff
@@ -24,8 +24,28 @@ module.exports = {
         noUselessIndex: true,
       },
     ],
+    'no-unneeded-ternary': [
```

Comment:
we could remove that rule, cause is webstorm solve that issue

Reply:
I removed this rule, but webstorm is only one IDE, many peoples don't use webstorm.